hack/validate/vendor

```diff
@@ -40,7 +40,7 @@ validate_vendor_diff(){
 validate_vendor_used() {
 	for f in $(mawk '/^[a-zA-Z0-9]/ { print $1 }' vendor.conf); do
 	if [ -d "vendor/$f" ]; then
-		found=$(echo "vendor/$f/"* | grep -iEc '/(LICENSE|COPYING)')
+		found=$(echo "vendor/$f/"* | grep -iEc '/(LICENSE|COPYING)' || true)
 		if [ "$found" -eq 0 ]; then
```

Comment:
If we're checking the output of `grep` and that's all this is for, then we should just check the return value directly instead, ala:

```bash
if ! echo ... | grep -qiEc ...; then
```

(Sorry for brevity, on my phone but I think you get the idea)